src/nvim/CMakeLists.txt

```diff
@@ -466,6 +466,13 @@ add_custom_target(update_version_stamp
     -P ${PROJECT_SOURCE_DIR}/cmake/GenerateVersion.cmake
   BYPRODUCTS ${NVIM_VERSION_GIT_H})
 
+set(NVIM_VERSION_DEF_H ${PROJECT_BINARY_DIR}/cmake.config/auto/versiondef.h)
+add_custom_command(
+  OUTPUT "${NVIM_VERSION_DEF_H}"
+  COMMAND "${CMAKE_COMMAND}"
+  ARGS -E copy "${PROJECT_BINARY_DIR}/cmake.config/auto/versiondef-$<CONFIG>.h" "${NVIM_VERSION_DEF_H}"
```

Comment:
Delete `ARGS`, it's legacy.

Reply:
Removed 548c99e0d075662ec06abd2e6c1f65ddfde961d8